content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/shipments_deliveries.rst

```diff
@@ -2,41 +2,49 @@
    - `Push & Pull Rules (eLearning Tutorial)
-     <https://www.odoo.com/slides/slide/push-pull-rules-1024>`_
+     <https://www.odoo.com/slides/slide/push-pull-rules-5789>`_
+
+One, two, and three-step receipts and deliveries
+================================================
+
+The default incoming route and outgoing route in Odoo is to receive goods and deliver goods in one
+step. However, once the *Multi-Step Routes* feature is enabled, the incoming and outgoing routes can
+be changed to either two steps or three steps, also.
 
-Choosing the right inventory flow to handle receipts and deliveries
-===================================================================
+Adding additional steps to receipts and deliveries moving in and out of the warehouse adds
+additional operations that must be completed *before* a product is finally received or shipped.
 
-By default, Odoo handles shipping and receiving in three different ways: in one, two, or three
-steps. The simplest configuration is one step, which is the default. Each additional step required
-for a warehouse for either the receiving or shipping process will add an additional layer of
-operations to perform before a product is either received or shipped. These configurations depend
-entirely on the requirements for the products stored, such as performing quality checks on received
-products, or using special packaging on shipped products.
+These configurations vary depending on the needs of the business, and depend on any requirements for
+the products stored, including performing quality control checks on received products, or using
+special packaging on shipped products.
 
 One-step flow
 -------------
 
 The receiving and shipping rules for a one-step configuration are as follows:
 
-- **Receipt**: Receive products directly into stock. No intermediate steps between receipt and stock
-  occur, such as a transfer to a quality control location.
-- **Shipping**: Ship products directly from stock. No intermediate steps between stock and shipping
-  occur, such as a transfer to a packing location.
-- Can only be used if not using :abbr:`FIFO (First In, First Out)`, :abbr:`LIFO (Last In, First
-  Out)`, or :abbr:`FEFO (First Expired, First Out)` removal strategies.
-- Receipts and/or deliveries are handled quickly.
-- Recommended for small warehouses with low stock levels, and for non-perishable items.
+- **Receipt**: receive products directly from vendors into stock. *No* intermediate steps between
+  receipt and input to stock occur, such as a transfer to a quality control location.
+- **Shipping**: ship products directly from warehouse stock to customer. *No* intermediate steps
+  occur before shipping, such as a transfer to a packing location.
+- Can *only* be used if **not** using :abbr:`FIFO (First In, First Out)`, :abbr:`LIFO (Last In,
+  First Out)`, or :abbr:`FEFO (First Expired, First Out)` removal strategies.
```

Comment:
the acronyms of these removal strategies have yet to be defined on this doc o; I'd spell these out first, in this line (:

Reply:
Nice catch. Suggestion implemented ✅